Dockerfile

```diff
@@ -4,7 +4,7 @@ COPY . .
 RUN go build -o machine-controller-manager ./cmd/manager
 RUN go build -o manager ./vendor/github.com/openshift/cluster-api/cmd/manager
 
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
For me it's simply that the current image does not work any more since yesterday:

```
http://base-4-0.ocp.svc/rhel-fast-datapath/repodata/repomd.xml: [Errno 14] curl#6 - "Could not resolve host: base-4-0.ocp.svc; Unknown error"
Trying other mirror.
 One of the configured repositories failed (rhel-fast-datapath),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:
     1. Contact the upstream for the repository and get them to fix the problem.
     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).
     3. Run the command with the repository temporarily disabled
            yum --disablerepo=rhel-fast-datapath ...
     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:
            yum-config-manager --disable rhel-fast-datapath
        or
            subscription-manager repos --disable=rhel-fast-datapath
     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:
            yum-config-manager --save --setopt=rhel-fast-datapath.skip_if_unavailable=true
failure: repodata/repomd.xml from rhel-fast-datapath: [Errno 256] No more mirrors to try.
http://base-4-0.ocp.svc/rhel-fast-datapath/repodata/repomd.xml: [Errno 14] curl#6 - "Could not resolve host: base-4-0.ocp.svc; Unknown error"
The command '/bin/sh -c INSTALL_PKGS="       libvirt-libs openssh-clients genisoimage       " &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all' returned a non-zero code: 1
```

combined with the fact that I was told by @praveenkumar that this is known to not work and he's always used the image i'm replacing it with.

Reply:
`Dockerfile` and `Dockerfile.rhel` have different needs to be able to built. At the moment `Dockerfile.rhel` uses `FROM registry.svc.ci.openshift.org/ocp/4.0:base` which is clearly downstream. But why is upstream not able to compile using 'public' images?

Reply:
@gbraad The image is public, but repos that it uses are RHEL one with the link `http://base-4-0.ocp.svc/` that looks like some private repository.